### PR TITLE
Topology feedback requires orchestrator refactoring

### PR DESCRIPTION
**Pull request by nakomis**

### CODE REVIEW

**Comment by sjcorbett** [comment]
@nakomis Could you rebase on master please?

File: src/main/javac/org/apache/brooklyn/tosca/a4c/brooklyn/ToscaComputeToVanillaConverter.java [hunk 1]
```diff
@@ -50,6 +50,7 @@ public ToscaComputeToVanillaConverter(ManagementContext mgmt) {
         spec.configure(VanillaSoftwareProcess.LAUNCH_COMMAND, "true");
         spec.configure(VanillaSoftwareProcess.STOP_COMMAND, "true");
         spec.configure(VanillaSoftwareProcess.CHECK_RUNNING_COMMAND, "true");
+        spec.configure("tosca.template.id", id);
```
Comment (sjcorbett) [inline src/main/javac/org/apache/brooklyn/tosca/a4c/brooklyn/ToscaComputeToVanillaConverter.java:53]: Cleaner to make tosca.template.id a config key and use it here and in ToscaPlanToSpecTransformer.

File: src/main/javac/org/apache/brooklyn/tosca/a4c/brooklyn/AlienApplicationModelUpdater.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+public class AlienApplicationModelUpdater {
+
+    private final ManagementContext mgmt;
+
+    public AlienApplicationModelUpdater(ManagementContext mgmt) {
+        this.mgmt = mgmt;
+    }
+
+    public void writeToAlien(Entity entity) {
```
Comment (sjcorbett) [inline src/main/javac/org/apache/brooklyn/tosca/a4c/brooklyn/AlienApplicationModelUpdater.java:33]: Does this class have any purpose over manual testing? Is it worth including? Can it be transformed into a proper test?

**Comment by nakomis** [comment]
Will rebase, add the ConfigKey you suggested, and (for now) remove AlienApplicationModelUpdater. The intention is that AlienApplicationModelUpdater will be called by Brooklyn, but don't yet have a suitable hook for it to be called. Will move onto a separate branch

**Comment by nakomis** [comment]
Closing in favour of https://github.com/cloudsoft/brooklyn-tosca/pull/11

**Pull request closed without merging**